### ISSUE TITLE
Section -> Template Content Area -> Section - parent section settings apply to child

### ISSUE THREAD

**Issue by orhanbilikvar** [soflyy/breakdance-bugs#934]
Clarification from Louis:

Create a single post template. Add a section, and put a Template Content Area inside it. Then edit a single post with Breakdance and add a section.

-----

Original Post:

The section settings we use in Single Template affect the section-containers we use in the page.
I am solving it with code as below, but it may be confusing for many users.

article > .section-container {
  width: 100%!important;
  max-width: none!important;
}

BD 1.4

![section-conta](https://github.com/soflyy/breakdance-bugs/assets/48365764/29fa7cab-86c9-41df-a2f0-c27e85786895)

**Comment by louisreingold** [comment]
Yes. The global section settings apply globally. Not a bug.

**Comment by louisreingold** [comment]
sorry, please disregard my previous comment

**Comment by flowww-dev** [comment]
yes, the section should always use .section > .section-container
